### ISSUE TITLE
Automatically Labeling and Closing Stale Issues

### ISSUE THREAD

**Issue by maxcapraro** [InnerSourceCommons/InnerSourcePatterns#137]
@spier noted very correctly that there are stale issues and PRs (e.g. here https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/17#issuecomment-604108188).

This increases the burden for people who want to learn "what's going on" and makes it more difficult to new contributors. Also, it might give the impression that we don't care about our "project hygiene" :-) 

I suggest we enable Github actions that label and close stale issues and PRs.

Do you have objections / concerns / comments?

**Comment by maxcapraro** [comment]
(Some interesting discussion also going on here: https://innersourcecommons.slack.com/archives/C2EFRTS6A/p1585222759001800?thread_ts=1585222105.001600&cid=C2EFRTS6A)

**Comment by spier** [comment]
@maxcapraro where did you land on this one? Do you still want to do some automation to clearly indicate "staleness"?  Or do you want to do this manually for now?

Trying to prevent this issue here from becoming stale itself :)

**Comment by maxcapraro** [comment]
I looked into the ready-made Github actions for this. @lenucksi did so, too. We found that they ignore some actions in PRs / issues. That is problematic because it leads to issues / PRs being marked as stale or even closed, despite there being activity.

@lenucksi, you proposed to use `stalebot` as an alternative? Would that be something that we can tie in with a few lines? Otherwise, my proposal would be to ignore this issue for now (risking for it to become stale 😃) and focus on the content work.

**Comment by lenucksi** [comment]
> @lenucksi, you proposed to use `stalebot` as an alternative? Would that be something that we can tie in with a few lines? Otherwise, my proposal would be to ignore this issue for now (risking for it to become stale ) and focus on the content work.

It's a GitHub action / bot thing: https://github.com/apps/stale
Integrating it is a few lines. The issues around it are likely similar to those that @maxcapraro  described. I'm happy to try it out, but we should configure it to only bring up friendly reminders not close things. 
And add a note that stale-bot can be drunk and hence isn't allowed to make final decisions, only TCs are and hence should be tagged if stale-bot appears to be drunk. ;)

In other words: 
- I'd consider a PR, feel welcome to create one.
- We need to monitor what stale-bot does.
- If stale-bot manages to collect twelve points for DUI, let's remove it. (For non-Germans: DUI and other traffic-related offenses are recorded in a register in a city called Flensburg and if you manage to collect twelve points you've had a drivers license for the longest time...)

**Comment by spier** [comment]
I am opting against doing anything automatic for now, as the number of issues/PRs is fairly limited.

Instead we should rather try to engage with the people that opening PRs/issues more frequently on a human-to-human level, to see if we can help them see their things through to completion.

Closing this issue but of course happy to revisit this as things change.
